Analysis/src/TauUncertainties.cpp

```diff
@@ -84,7 +84,7 @@ double TauESUncertainties::GetCorrectionFactorTrueTau(analysis::Period period, i
     if(pt > 400)
         correction_factor = 1 + static_cast<int>(current_scale) * 0.03;
```

Comment:
- remove variable `correction_factor`, use `tau_correction`

Reply:
but that returns a double and tau_correction is PhysicalValue, is it possible that equality?

Reply:
`tau_correction = PhysicalValue(0., 3.)` to be compatible with other definitions in the map.